### ISSUE TITLE
Qt errors

### ISSUE THREAD

**Issue by jlsam** [Lindacoin/Linda#53]
I'm getting these errors in debug.log, when using the Qt wallet in Ubuntu, by clicking the different tabs on the left side:

> GUI: QCoreApplication::postEvent: Unexpected null receiver
> GUI: QCoreApplication::postEvent: Unexpected null receiver
> GUI: QCoreApplication::postEvent: Unexpected null receiver
> GUI: QObject::connect: No such signal WalletModel::balanceChanged(qint64, qint64, qint64, qint64)
> GUI: QObject::connect:  (receiver name: 'SendCoinsDialog')
> GUI: QCoreApplication::postEvent: Unexpected null receiver
> GUI: QCoreApplication::postEvent: Unexpected null receiver

Also most of the time I cannot see my masternode listed in the wallet, it's just empty. But I cannot replicate this, it's just there sometimes, whereas the errors are consistent. I'm not sure it could be related to this issue. The system is updated.

I have downloaded and compiled the Qt wallet yesterday again, but the issue remains.

**Comment by Chris-Lindacoin** [comment]
Hi jlsam, I've seen youve posted that this is sorted now, can you please close if you are all good, or reach out to me chris@lindacoin.com for further assistance

**Comment by jlsam** [comment]
Actually, I just tested and the GUI errors are still here. Can you not see them in your log?

**Comment by Chris-Lindacoin** [comment]
Which errors are you getting?

Cheers
Chris

**Comment by jlsam** [comment]
Which errors?... Well, the same ones I indicated when I opened this Issue.

**Comment by Chris-Lindacoin** [comment]
Hey can you see your masternode when you type in the debug console “masternode list”?

**Comment by jlsam** [comment]
The errors appear in debug.log when I see my masternode in the list and when I don't. That doesn't seem to matter. I'm running a cold/remote MN setup, where a GUI wallet in Ubuntu controls remotely Lindad running as masternode also in a VPS running Ubuntu.

The errors first appeared when I was running the local wallet in Ubuntu 14, but I have since upgraded to Ubuntu 18 and the same errors persist.

What is strange is that I'm receiving MN rewards although the coins never appear locked in my local wallet.

This is when I click 'Send'
GUI: libpng warning: iCCP: known incorrect sRGB profile
GUI: QObject::connect: No such signal WalletModel::balanceChanged(qint64, qint64, qint64, qint64)
GUI: QObject::connect:  (receiver name: 'SendCoinsDialog')

When I click any of the other options, I get just
GUI: QCoreApplication::postEvent: Unexpected null receiver

The layout in the wallet still appears correctly, it's just that these messages are being logged.
I also don't see a tab just for my masternodes, all I see is a tab for all masternodes.

**Comment by jlsam** [comment]
yes. and I receive rewards too28.05.2018, 13:56, "Chris-Lindacoin" <notifications@github.com>:Hey can you see your masternode when you type in the debug console “masternode list”?
